07-Modelling-Patchy-Particles/01-Kern-Frenkel-Model.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0",
```

Comment:
<p>Change</p><blockquote>Some systems are better represented by anisotropic pair potentials that depend on particle relative orientations. </blockquote><p>to</p><blockquote>Some systems are better represented by anisotropic pair potentials that depend on relative particle orientations. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>